packages/massa-proto-cli/src/ts-gen.ts

```diff
@@ -0,0 +1,21 @@
+import { ProtoFile } from './protobuf';
+
+/**
+ * Generate the TypeScript code for the ts caller function
+ * to serialize the arguments
+ *
+ * @param protoFile - The protoFile object
+ * @returns - The generated serialization code
+ */
+function argumentSerialization(protoFile: ProtoFile): string {
+  let content = `
+  // Serialize the arguments
+  const serializedArgs = ${protoFile.funcName}Helper.toBinary({
+  `;
+  for (let arg of protoFile.argFields) {
+    content += `\t\t${arg.name}: ${arg.name},\n`;
+  }
+  content = content.slice(0, -2); // remove the last comma and line break
+  content += `\n\t});\n`;
+  return content;
+}
```

Comment:
note: I fixed what I think is a bug `\t\t${arg.name}: ${arg.name},\n` => `\t\t${arg.name}: ${arg.value},\n`, maybe i'm wrong.

```suggestion
function generateArgumentsSerialization(protoFile: ProtoFile): string {
  let args = "";
  for (const arg of protoFile.argFields) {
    args += `\t\t${arg.name}: ${arg.value},\n`;
  }
  
  return `
  // Serialize the arguments
  const serializedArgs = ${protoFile.funcName}Helper.toBinary({
      ${args.slice(2,-2)}
  });`;
}
```

Reply:
yes it was a bug. I fixed it. ty